server/src/routes/api/v1/dogParksRouter.js

```diff
@@ -44,4 +43,25 @@ dogParksRouter.get("/:id", async (req, res) => {
     }
 })
 
+dogParksRouter.post("/:id", async (req, res) => {  
```

Comment:
this requires a nested router, like in week 6
https://learn.launchacademy.com/teams/admins/curricula/on-campus-boston-40/lesson_groups/week_6:_database_associations_in_monolith_apps/lessons/persisting-related-records-in-objection-and-express